app/Services/Geocoder/PositionstackGeocoder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services\Geocoder;
+
+use Illuminate\Support\Facades\Http;
+
+class PositionstackGeocoder implements GeocoderInterface {
+    public function geocode(string $address): ?array {
+        $response = Http::get('http://api.positionstack.com/v1/forward', [
+            'access_key' => config('positionstack.api_key'),
+            'query' => $address,
+            'country' => 'NL',  // TODO allow overriding?
+            'limit' => 1
+        ]);
+
+        if ($response->successful()) {
```

Comment:
Try to keep the happy path on the same indent level en use if statements for exceptions. In this case you would need to reverse the if statement.
Also why not throw an error on an unsuccessfull response? This will lead to a silent failure, which isn't ideal. I tend to throw from deep within implementation details and only catch/silence them more to the surface (like controllers and commands)